pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiTemplate.java

```diff
@@ -11,25 +13,35 @@
 import org.jdbi.v3.sqlobject.config.RegisterConstructorMapper;
 import org.jdbi.v3.sqlobject.customizer.Bind;
 import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlBatch;
 import org.jdbi.v3.sqlobject.statement.SqlQuery;
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 
 public interface JdbiTemplate extends SqlObject {
 
+    String INSERT_TEMPLATE_SQL = "INSERT INTO template(template_code, template_type_id, template_text, revision_id) "
+            + "SELECT :templateCode, type.template_type_id, :templateText, :revisionId "
+            + "FROM template_type AS type "
+            + "WHERE type.template_type_code = :templateType";
+
```

Comment:
SQL is unchanged from original, but moved it so can be shared between batch and non-batch versions of insert method.